### ISSUE TITLE
Please add the ability to disable fish and use bash from as a choice in the installer

### ISSUE THREAD

**Issue by ryzendew** [CachyOS/linux-cachyos#269]
Look i hate fish sooooo much it's weird and clunky and sets off my mental state can you please let us use bash as a choice in the install cause unpicking fish does nothing! please i beg of you ... i don't want to suffer any longer.

Sent with love from the youtuber that whores you out Daily mattscreative

**Comment by EnderSpy29** [comment]
Yes! add an option in the installer to use a different shell, or just use Bash. i don't want fish on my system. unpicking it should actually do something at least.

**Comment by praetorxyn** [comment]
I use fish, but I think it would be better to have one of the installer steps being selecting the preferred shell, and just having one of them selected by default (whether it's bash for mass appeal and familiarity or fish because it's likely the best for new users).

**Comment by EnderSpy29** [comment]
> I use fish, but I think it would be better to have one of the installer steps being selecting the preferred shell, and just having one of them selected by default (whether it's bash for mass appeal and familiarity or fish because it's likely the best for new users).

should be Bash. Bash just works. on every system everywhere
